lib/cloud_controller/opi/stager_client.rb

```diff
@@ -57,6 +58,11 @@ def staging_completion_callback(staging_details)
       "#{scheme}://#{auth}@#{host_port}#{path}"
     end
 
+    def build_env(environment)
+      env = OPI.muse(environment)
```

Comment:
Not complaining about the method name, I realize you're just adhering to the interface of the [env var philosopher](https://github.com/cloudfoundry/cloud_controller_ng/blob/9ea859e186f2a727663e0e9e75b5baacd83592f6/lib/cloud_controller/diego/normal_env_hash_to_diego_env_array_philosopher.rb#L6). 😂 

But if feels strange to have the method directly on the `OPI` module like that. Can we make it `OPI::EnvHash.muse` or something?